tests/python/gaia-ui-tests/gaiatest/apps/phone/app.py

```diff
@@ -10,6 +10,7 @@
 class Phone(Base):
 
     name = "Phone"
```

Comment:
There's no difference in python, it's just a matter of style and to be able to include quotes in strings. For instance:
``` python
string = 'That is a "quoted" string'
string2 = "That's a 'quoted' string"
```